src/app/components/student-record-display/student-record-display.component.html

```diff
@@ -58,7 +58,7 @@ <h5 class="card-titile d-inline mx-2">Request History</h5>
                                 <table class="table table-striped">
                                     <thead>
                                         <tr>
-                                            <th scope="col">Token Option Name</th>
+                                            <th scope="col">Request&rsquo;s Name</th>
```

Comment:
Not sure should we use "Request's Name" here. The request itself does not have a name, but the token option being requested has. Since we need to have the concept of token options for instructors, maybe something like "Requested Token Option" is a better choice.